data/strategies/0x86eD4F77d40182b8686a25e125FB3f5a04203CaA.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyLenderYieldOptimiser",
+  "description": "This strategy lends the {{tokens}} on various lending platforms such as CREAM, AAVE or ALPHA HOMORA to gain yield.",
+  "author": "Sam Priestley",
+  "authorUri": "https://twitter.com/arbingsam"
+}
```

Comment:
are all this strategies from another PR?

Reply:
yes, sorry, just removed this commit from the PR